lib/stitch_plan/stitch.py

```diff
@@ -3,17 +3,21 @@
 # Copyright (c) 2010 Authors
 # Licensed under the GNU GPL version 3.0 or later.  See the file LICENSE for details.
 
+from typing import Union, Optional
 from shapely import geometry as shgeo
 
 from ..utils.geometry import Point
 
 
 class Stitch(Point):
     """A stitch is a Point with extra information telling how to sew it."""
+    x: float
+    y: float
 
     def __init__(
         self,
-        x, y=None,
+        x: Union["Stitch", shgeo.Point, float],
```

Comment:
I think we should say `"Point"` here instead of `"Stitch"`.  There may not be anywhere in the code that actually initializes a point from another point, but it seems like a reasonable thing to do, to copy a point.